post-service/src/main/resources/application-external.yml

```
@@ -19,4 +19,8 @@ spring:
       value-deserializer: org.springframework.kafka.support.serializer.JsonSerializer
     producer:
       key-serializer: org.apache.kafka.common.serialization.StringSerializer
-      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
\ No newline at end of file
+      value-serializer: org.springframework.kafka.support.serializer.JsonSerializer
+
+feign:
+  url:
+    organization: ${ORGANIZATION_CLIENT_URL}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)